src/undomanager.js

```diff
@@ -22,29 +22,36 @@ define([
         $('.fd-undo-container').append(UNDO_ALERT);
     }
 
-    function toggleAlert(undoStack) {
+    function toggleAlert(undoStack, vellum) {
         if (undoStack.length && !alertShown()) {
             createAlert();
         } else if (undoStack.length === 0 && alertShown()) {
-            $('.fd-undo-delete').alert('close');
+            $('.fd-undo-delete').remove();
+        }
+        if (vellum) {
+            vellum.adjustToWindow();
         }
     }
 
     function UndoManager() {
-        this.undoStack = [];
+        var _this = this;
+        _this.undoStack = [];
+        _this.vellum = undefined;
```

Comment:
yeah, i like that
https://github.com/dimagi/Vellum/pull/639/commits/13e30e30147eccd7ff6d6bd0a7e62e977f8bae3b